src/config/rest.js

```diff
@@ -20,6 +20,19 @@ export const adminEndpoint = !process.env.REACT_APP_ADMIN_END
     ? 'http://localhost:8000/admin/'
     : `${reactAppApiHttps}://${process.env.REACT_APP_ADMIN_END}/admin/`;
 
+export const serverlessEndpoint = (() => {
+    switch (process.env.REACT_APP_DEEP_ENVIRONMENT) {
```

Comment:
I think if the process.env.REACT_APP_SERVERLESS_DOMAIN is set, it should take precedence over all others